conf/iptables.conf.example

```diff
@@ -87,9 +98,6 @@
 # DNS
 -A input-internal-vlan-if --protocol tcp --match tcp --dport 53  --jump ACCEPT
 -A input-internal-vlan-if --protocol udp --match udp --dport 53  --jump ACCEPT
-# DHCP
--A input-internal-vlan-if --protocol udp --match udp --dport 67  --jump ACCEPT
```

Comment:
Shouldn't we still listen for those using a port bind? 
Or are you using libpcap for grabbing the packs?
Or did I get everything wrong?

Reply:
This rule is managed by iptables.pm